src/ray/object_manager/object_manager.cc

```diff
@@ -736,7 +736,11 @@ std::string ObjectManager::DebugString() const {
 void ObjectManager::RecordMetrics() {
   pull_manager_->RecordMetrics();
   push_manager_->RecordMetrics();
-  stats::ObjectStoreAvailableMemory().Record(config_.object_store_memory - used_memory_);
+  // used_memory_ includes the fallback allocation, so we should subtract it here
+  // to calculate the exact available memory.
+  stats::ObjectStoreAvailableMemory().Record(
+      config_.object_store_memory - used_memory_ -
```

Comment:
should it be `config_.object_store_memory - used_memory_ + plasma::plasma_store_runner->GetFallbackAllocated()` ?

Reply:
Yep it should be `config_.object_store_memory - used_memory_ - plasma::plasma_store_runner->GetFallbackAllocated()` or `config_.object_store_memory - (used_memory_ + plasma::plasma_store_runner->GetFallbackAllocated())`

Reply:
Yep it should be `config_.object_store_memory - used_memory_ - plasma::plasma_store_runner->GetFallbackAllocated()` or `config_.object_store_memory - (used_memory_ + plasma::plasma_store_runner->GetFallbackAllocated())`